rs/src/main.rs

```diff
@@ -0,0 +1,57 @@
+use anyhow::{Context, Result};
+use clap::{Parser, Subcommand};
+use std::fs;
+
+/// Simple CLI for Artisan Tools.
+#[derive(Parser)]
+#[command(name = "artisan-tools")]
+#[command(version = "0.1.0")]
```

Comment:
No need for writing the version, it will take it from `Cargo.toml` automagically
```suggestion
```